vulns/mistune/PYSEC-2022-237.yaml

```diff
@@ -18,25 +18,6 @@ affected:
     - introduced: "0"
```

Comment:
Thanks @lepture!  In order for this to accomplish what you're wanting you'll also want to set this:
```suggestion
    - introduced: "2.0.0a1"
```

The versions array is calculated from the affected range, so if you don't update that it'll just add in the versions you removed on the next run of the analysis job